sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/google-recaptcha.yaml

```
@@ -23,10 +23,10 @@ provider:
           # False Positives: ReCaptcha offificial test keys:
           # Site key: *_MXjiZKhI
           # Secret key: *J4WifJWe
-          patternNot: "(\
-            x{6,}|\
-            6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI|\
-            6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe)"
+          patternNot:
+            - "x{6,}"
+            - "6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI"
+            - "6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe"
```

```suggestion
            - "(\\w)\\1{6,}"
            # ReCaptcha official testing site key
            - "6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI"
            # ReCaptcha official testing secret key
            - "6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe"
```

Can you remove the comments between the previous module and there ? 🙏